main/src/cgeo/geocaching/connector/oc/OkapiClient.java

```diff
@@ -329,6 +330,12 @@ private static void fillForBasicFilter(final BaseGeocacheFilter basicFilter, fin
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).difficultyGeocacheFilter, params, valueMap, connector);
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).terrainGeocacheFilter, params, valueMap, connector);
                 break;
+            case RATING:
+                final RatingGeocacheFilter ratingFilter = (RatingGeocacheFilter) basicFilter;
+                if (ratingFilter.getMinRangeValue() != null) {
+                    valueMap.put("rating", (ratingFilter.getMinRangeValue() == null ? "1" : ((int) Math.floor(ratingFilter.getMinRangeValue()))) + "-" + (ratingFilter.getMaxRangeValue() == null ? "5" : Math.round(ratingFilter.getMaxRangeValue())));
+                }
+                break;
```

Comment:
Exactly. And don't forget to make ISearchByFilter.getFilterCapabilities also return the new RATING type in its result

Reply:
Have added it, thanks